### PR TITLE
adding duplicated args/params validation

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/15683

### CODE REVIEW

**Comment by ronykoz** [comment]
@anara123 forgot that, added it now.
Also added proper message for the image(After the Content-team discussion about it earlier today)

File: Tests/scripts/hook_validations/image.py [hunk 1]
```diff
@@ -20,10 +20,18 @@ def __init__(self, file_path):
             self.file_path = file_path
         else:
             if re.match(INTEGRATION_YML_REGEX, file_path, re.IGNORECASE):
-                self.file_path = glob.glob(os.path.join(os.path.dirname(file_path), '*.png'))[0]
+                try:
+                    self.file_path = glob.glob(os.path.join(os.path.dirname(file_path), '*.png'))[0]
+                except IndexError:
+                    self._is_valid = False
+                    print("You've created/modified a package but failed to provide an image as a .png file, "
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/image.py]: use print_error for all errors
```suggestion
                    print_error("You've created/modified a package but failed to provide an image as a .png file, "
```

File: Tests/scripts/hook_validations/integration.py [hunk 2]
```diff
@@ -46,9 +46,52 @@ def is_backward_compatible(self):
...
+        Returns:
+            bool. True if there are duplicates, False otherwise.
+        """
+        commands = self.current_integration.get('script', {}).get('commands', [])
+        for command in commands:
+            arg_list = []
+            for arg in command['arguments']:
+                if arg in arg_list:
+                    self._is_valid = False
+                    print("The argument '{}' of the command '{}' is duplicated in the integration '{}', "
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/integration.py]:
```suggestion
                    print_error("The argument '{}' of the command '{}' is duplicated in the integration '{}', "
```

File: Tests/scripts/hook_validations/integration.py [hunk 3]
```diff
@@ -46,9 +46,52 @@ def is_backward_compatible(self):
...
+        Returns:
+            bool. True if there are duplicates, False otherwise.
+        """
+        configurations = self.current_integration.get('configuration', [])
+        param_list = []
+        for configuration_param in configurations:
+            param_name = configuration_param['name']
+            if param_name in param_list:
+                self._is_valid = False
+                print("The parameter '{}' of the integration '{}' is duplicated, please remove one of its appearances "
```
Comment (yaakovi) [inline Tests/scripts/hook_validations/integration.py]:
```suggestion
                print_error("The parameter '{}' of the integration '{}' is duplicated, please remove one of its appearances "
```

**Pull request merged**